unic/char/range/src/step.rs

```diff
@@ -0,0 +1,40 @@
+use std::char;
```

Comment:
Since we have a module here called `char`, I think it would be easier to read the code if we stick with `std::char` instead of using it as `char`. What do you think?